goto/tests/test_end_to_end.sh

```diff
@@ -217,26 +217,57 @@ function test_06_goto_add {
     _projectfile_should_contain "test_escaped_ampersand_in_url"
 }
 
-function test_07_goto {
-    magicword="test_goto"
-    testcdpath="/tmp"
-    nonexisting_magicword="IDoNotExist"
-    uri="http://example.com"
+function test_07_goto_many {
+    declare -a magicwords
+    declare -a uris
+    magicwords=(
+        "red"
+        "blue"
+        "green"
+        "yellow"
+        "orange"
+    )
+    uris=(
+        "\"http://red.com\""
+        "\"mailto:jon@gmail.com\""
+        "\"spotify:green.com\""
+        "\"sftp:hallo@yellow.com\""
+        "\"goto+subl://orange.com\""
+    )
+
+    for i in "${!magicwords[@]}"; do
```

Comment:
The use of these for loops is clean!

Reply:
I agree, if you don't consider the monster in the middle which is `"${!magicwords[@]}"`

Reply:
"bash-clean"

Reply:
Hehe - well goto is a project that could level up bash skills.

Reply:
https://www.linuxjournal.com/content/bash-arrays

Reply:
Thats a really good resource. Best I have seen. Always have to google how the bash arrays work. They are a bit .... different